bin/inference/pycbc_inference

```diff
@@ -223,7 +272,8 @@ with ctx:
                         psds=psd_dict, prior=prior_eval,
                         sampling_parameters=sampling_parameters,
                         replace_parameters=replace_parameters,
-                        sampling_transforms=sampling_transforms)
+                        sampling_transforms=sampling_transforms,
+                        transfer_functions=tf_dict, calib=opts.calib)
```

Comment:
No need to supply a ``calib`` option - if the other calibration options aren't there, then you should be able to do the logic inside the class to realize don't use them, eg. set the default of ``transfer_functions`` to ``None`` and if its ``None`` then don't use calibration.

Though instead of passing the transfer_functions, pass it a ``Relcalibrate`` instance - don't construct the calibration in the likelihood class.